cirq-google/cirq_google/devices/google_noise_properties.py

```diff
@@ -66,14 +83,101 @@ class GoogleNoiseProperties(devices.SuperconductingQubitsNoiseProperties):
             for that gate. Defaults to no-op for all gates.
     """
 
-    fsim_errors: Dict[noise_utils.OpIdentifier, cirq.PhasedFSimGate] = field(default_factory=dict)
+    fsim_errors: Dict[noise_utils.OpIdentifier, cirq.PhasedFSimGate] = dataclasses.field(
+        default_factory=dict
+    )
 
     def __post_init__(self):
         super().__post_init__()
 
         # validate two qubit gate errors.
         self._validate_symmetric_errors('fsim_errors')
 
+    def override(
```

Comment:
Could we rename this to something more informative like `update_parameters` or something, just so it indicates that we are updating/setting certain properties with this function ?

Reply:
Opted for `with_params`, since `with_*` is used in many places in Cirq to mean "create a copy of this object, but with the given modifications".